incubator/webpagetest-server/values.yaml

```diff
@@ -0,0 +1,85 @@
+# Default values for wpt-test.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: webpagetest/server
+  tag: latest
+  pullPolicy: IfNotPresent
+service:
+  name: webpagetest-server
+  type: ClusterIP
+  externalPort: 80
+  internalPort: 80
+
+# Front end for users.
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  hosts:
+    - web-page-test.local
+  # It's best to add annotations to restrict who can access
+  annotations:
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+    # ingress.kubernetes.io/force-ssl-redirect: "true"
+    # ingress.kubernetes.io/whitelist-source-range: '127.0.0.1/32,192.168.0.0/24'
+  tls:
+    # Secrets must be manually created in the namespace.
```

Comment:
If you look at the Wordpress chart you'll see a pattern for passing in or letting Helm create the secret for you. Could you take a look at that pattern?

Reply:
I did a `helm create`, then copied the readme from the chart I had previously made a pull request on. 
Is the issue 
* I've added more example ingress annotations. I can remove those, however they are provided as an example due to the potential cost issue with using AMI's
* Is it the structure of the tls section. This is what `helm create` produces as a template. 
If it's the latter I think the issue should be raised in the helm project so that the `helm create` command produces the desired starting point.